tests/dist/chats.spec.dev.js

```diff
@@ -15,7 +18,8 @@ var token;
 var loggedInUserId;
 beforeAll(function _callee() {
   return regeneratorRuntime.async(function _callee$(_context) {
-    while (1) {
+    const condition = 1;
```

Comment:
Is it possible for this constant to be global for this test suite, so you only declare it once?

I've seen it repeat quite a few times throughout this file